sechub-schedule/src/test/java/com/mercedesbenz/sechub/domain/schedule/strategy/SchedulerStrategyFactoryTest.java

```diff
@@ -6,33 +6,36 @@
 
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
-import org.springframework.test.util.ReflectionTestUtils;
 
 class SchedulerStrategyFactoryTest {
 
     private static final String FIFO_STRATEGY_ID = "first-come-first-serve";
     private static final String OOSP_STRATEGY_ID = "only-one-scan-per-project-at-a-time";
+    private static final String OOSPAG_STRATEGY_ID = "only-one-scan-per-project-and-module-group";
```

Comment:
It is difficult to differentiate between `OOSP` and `OOSPAG`. Furthermore, I would argue that `_STRATEGY_ID` is not the most important part of the name.

My suggestion, is to call OOSP: `ONLY_ONE_SCAN_PER_PROJECT`. And skip the `_STRATEGY_ID` part. The same for  `OOSPAG`.